### DISCUSSION TITLE
griefpreventions pvp arena

### DISCUSSION THREAD

**Discussion by Pupsaran** [GriefPrevention/GriefPrevention#2097] [Q&A]
Hi so i have this issue where i have a spawn, under the spawn i have created a pvp arena, the spawn itself is protected with worldedit + adminclaim. Now i have used worldedit to make a pvp arena before (having to mess with priorities and stuff) i have it all setup as it should be but it keeps saying ''You can not fight someone while you're protected from PvP'' <- that should be griefpreventions message. How do i allow pvp from griefpreventions side?

**Comment by bigpresh** [comment]
You should be able to set `PvP.ProtectFreshSpawns` in your GP config to disable that, or you can grant the `griefprevention.nopvpimmunity` permission via your permissions plugin to your default player class.

If you don't want GP's PvP protection at all, then look at `PvP.RulesEnabledInWorld` in your config and toggle `true` to `false` for worlds you don't want GP's PvP protection active in.

More details at https://docs.griefprevention.com/configuration/#pvp-configuration

**Reply by Pupsaran** [reply]
Thank you, got it to work now! :)
